docs/cmdline-opts/etag-save.d

```diff
@@ -0,0 +1,15 @@
+Long: etag-save
+Arg: <file>
+Help: Parse ETag from a request and save it to a file
+Protocols: HTTP
+---
+This option let's you save an HTTP ETag to specified file. Etag is
```

Comment:
"This option saves an HTTP ETag to the specified file. An Etag is a passed in the response headers."

Reply:
done